Web/Cookie.hs

```diff
@@ -90,7 +89,7 @@ parseCookie s =
 
 breakDiscard :: Word8 -> S.ByteString -> (S.ByteString, S.ByteString)
 breakDiscard w s =
-    let (x, y) = S.breakByte w s
+    let (x, y) = S.break (== w) s
```

Comment:
`breakByte` is deprecated, and `break` is faster; https://www.stackage.org/haddock/lts-9.13/bytestring-0.10.8.1/Data-ByteString.html#v:breakByte